### PR TITLE
kubernetes-cli 1.7.1

### PR DESCRIPTION
**Pull request by cblecker**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by ilovezfs** [comment]
@cblecker just noticed this ... why is the zsh completion file named kubectl not _kubectl?

**Comment by cblecker** [comment]
I never noticed that! Fixing..

**Comment by cblecker** [comment]
@ilovezfs Fixed! PTAL

File: Formula/kubernetes-cli.rb [hunk 1]
```diff
@@ -36,7 +36,12 @@ def install
 
       # Install zsh completion
       output = Utils.popen_read("#{bin}/kubectl completion zsh")
-      (zsh_completion/"kubectl").write output
+      (zsh_completion/"_kubectl").write output
+
+      # Install man pages
+      # Leave step this for the end as this dirties the git tree
```
Comment (ilovezfs) [inline Formula/kubernetes-cli.rb]: Interesting word order …

Comment (cblecker) [inline Formula/kubernetes-cli.rb]: *facepalm*

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks @cblecker! :control_knobs:

**Pull request closed without merging**